xbmc/addons/AddonManager.cpp

```diff
@@ -22,6 +22,7 @@
 #include "filesystem/Directory.h"
 #include "filesystem/File.h"
 #include "filesystem/SpecialProtocol.h"
+#include "messaging/helpers/DialogOKHelper.h"
```

Comment:
I don't know the architecture of the addon subsystem, but it seems you are polluting a core component with GUI stuff here, which feels like bad design.

Reply:
ok, need to shift the dialog somewhere else..

Reply:
damn, that was a leftover, the dialog already moved to ` xbmc/addons/AddonSystemSettings.cpp`